introduction.md

```diff
@@ -6,3 +6,10 @@ Hi, I am Davy Van Roy.
 Please see: [davyvanroy.be](https://davyvanroy.be)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?